lib/internal/child_process.js

```diff
@@ -652,7 +649,8 @@ function setupChannel(target, channel, serializationMode) {
         }
       }
 
-      assert(ArrayIsArray(target._handleQueue));
+      validateArray(target._handleQueue, 'target._handleQueue');
```

Comment:
This does not seem useful. Assertions are for catching internal errors, validators are for catching invalid inputs. As far as I can tell, this condition relates to internal state and not to user input.